source/core.py

```diff
@@ -250,6 +250,48 @@ def onEndSession(evt):
 	import windowUtils
 	class MessageWindow(windowUtils.CustomWindow):
 		className = u"wxWindowClassNR"
+		batteryCache = None

```

Comment:
Is there a more descriptive name that more clearly states what this stores; e.g. lastBatteryStatus or similar?